### PR TITLE
[TRAFODION-2658] pyinstaller - support elastic adding nodes

### PR DESCRIPTION
**Pull request by mkby**
init version of pyinstaller elastic feature.

### CODE REVIEW

File: core/sqf/sql/scripts/sqregen [hunk 1]
```diff
@@ -41,8 +41,15 @@ function Usage {
 
 function RegenConfig {
     echo "Regenerating Trafodion configuration: $SQCONFIG_FILE"
+    cd $TRAF_HOME/sql/scripts
     ./regensq.pl $SQCONFIG_FILE
-    exit $?;
+    if [ $? != 0 ]; then
+        echo "Failed to run regensq.pl."
```
Comment (DaveBirdsall) [inline core/sqf/sql/scripts/sqregen]: The error message is incorrect. We did run regensq.pl. The problem is that regensq.pl failed.

File: core/sqf/sql/scripts/sqregen [hunk 2]
```diff
@@ -41,8 +41,15 @@ function Usage {
...
+        exit $?
+    fi
+    echo "Copying Trafodion configuration file $SQCONFIG_FILE to all nodes"
+    pdcp `trafconf -wname` $SQCONFIG_FILE $TRAF_HOME/sql/scripts/
```
Comment (DaveBirdsall) [inline core/sqf/sql/scripts/sqregen]: Should there be an error message if pdcp fails?

File: core/sqf/sql/scripts/sqregen (hunk 1 above)
Comment (mkby) [inline core/sqf/sql/scripts/sqregen]: sounds good, I will change the message, thanks.

File: core/sqf/sql/scripts/sqregen (hunk 2 above)
Comment (mkby) [inline core/sqf/sql/scripts/sqregen]: I think pdcp itself will show the error message if fails

**Comment by mkby** [comment]
jenkins, retest

**Pull request merged**